src/gui/dialogs/DialogStateful.hpp

```diff
@@ -80,12 +75,12 @@ class IDialogStateful : public IDialog {
     bool Change(uint8_t phs, uint8_t progress_tot, uint8_t progress); // = 0; todo should be pure virtual
     virtual ~IDialogStateful();
 
-    static constexpr rect_ui16_t get_radio_button_size() {
```

Comment:
constexpr  has meaning here, removing it will increase code size
you should refactor gui_defaults first to satisfy constexpr

Reply:
I made BFW-954 to split the issue. It's not crucial to do it immediately.